backend/beyond_the_loop/routers/models.py

```
@@ -135,11 +134,51 @@ async def toggle_model_by_id(id: str, user=Depends(get_verified_user)):
             detail=ERROR_MESSAGES.NOT_FOUND,
         )
 
+#################################
+# Add and remove from bookmarked
+#################################
+
+@router.post("/model/{model}/bookmark/update")
+async def update_model_bookmark(model: str, user=Depends(get_verified_user)):
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)